frontend/public/components/monitoring/targets.tsx

```diff
@@ -293,9 +320,34 @@ const REDUX_ID = 'monitoringTargets';
 
 const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.lastError });
 
+const getWatchError = (monitorsWatchContext, t) => {
+  const loadErrorIndex = 2;
+  const watchError = monitorsWatchContext[loadErrorIndex];
```

Comment:
```suggestion
  const [, , watchError] = monitorsWatchContext;
```

Probably easier to read and mirrors how we do this elsewhere.